Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardSettingsViewController.swift

```diff
@@ -52,13 +55,12 @@ class CreditCardSettingsViewController: UIViewController, Themeable {
                                                         toggleTextColor: theme.colors.textPrimary.color)
         let emptyView = CreditCardSettingsEmptyView(colors: colors, toggleModel: viewModel.toggleModel)
         self.creditCardEmptyView = UIHostingController(rootView: emptyView)
-
-        let creditCardEditView = CreditCardEditView(viewModel: viewModel.addEditViewModel,
-                                                    removeButtonColor: themeManager.currentTheme.colors.textWarning.color,
-                                                    borderColor: themeManager.currentTheme.colors.borderPrimary.color)
-        self.creditCardAddEditView = SensitiveHostingController(rootView: creditCardEditView)
```

Comment:
I removed the SensitiveHostingController as we will move it to the `CreditCardSettingsViewController` to ideally authenticate user `once` instead of doing it all the time